tf2onnx/graph.py

```diff
@@ -1065,27 +1063,18 @@ def optimize_model_proto(onnx_model_proto):
         """Optimize the model proto, for example: eliminating all useless Transpose pairs.
 
         Returns:
-            model proto after optimization, if optimizer run successfully
```

Comment:
try-catch needs to be kept otherwise it will break the conversion if onnx infer shape failed.

Reply:
done